udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,144 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.bottomRow
+{
+	padding-top: 30px;
```

Comment:
Style prescribes 2-spaces for indentation:
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
This has been implemented in latest push.